fury/actor.py

```diff
@@ -559,7 +562,7 @@ def streamtube(lines, colors=None, opacity=1, linewidth=0.1, tube_sides=9,
     poly_mapper.Update()
 
     # Color Scale with a lookup table
-    if is_colormap:
+    if is_colormap is not None and is_colormap:
```

Comment:
I do not understand this change. Can you rename the variable or explain a bit more?

Reply:
renamed to "color_is_scalar" => can use colormap lut